### PR TITLE
Create Bylickilabs.json

### PR DESCRIPTION
**Pull request by bylickilabs**
resolves #issueNumber

## Checklist for profile addition

- [x] Name the PR `fix: add <Bylickilabs>`.
- [x] You've added file named `Bylickilabs.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by krshkun — APPROVED** [review]
Looks good! Thank you

File: profiles/Bylickilabs.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/109308073/209451382-c2c90f77-0c2a-45dc-bd06-1f5418910dd9.jpg",
+  "issueId": 1589,
```
Comment (Panquesito7) [inline profiles/Bylickilabs.json:3]: I believe the issue ID here must be #1588.
```suggestion
  "issueId": 1588,
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by bylickilabs — COMMENTED** [review]
Is it correct now? Please check again

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Review by bylickilabs — COMMENTED** [review]
Create on https://github.com/EddieHubCommunity/awesome-github-profiles
bylickilabs.json

**Pull request merged**